studio/src/editors/merch-card-editor.js

```diff
@@ -94,7 +94,8 @@ class MerchCardEditor extends LitElement {
         this.whatsIncludedElement
             ?.querySelectorAll('merch-mnemonic-list')
             .forEach((listEl) => {
-                const icon = listEl.querySelector('merch-icon')?.src || '';
+                const iconEl = listEl.querySelector('merch-icon');
+                const icon = iconEl?.src || iconEl?.getAttribute('src') || '';
```

Comment:
I wonder in which case `iconEl?.getAttribute('src')` is defined but `iconEl.src` is not.

Reply:
true, it is a bit redundant. I have simplified it.

Reply:
I was curious so I investigated it more.
Its because of this:
https://github.com/adobecom/mas/blob/main/studio/src/editors/merch-card-editor.js#L84

this element is not really coming from merch-card in DOM, not sure if expected or not, but that explains why WC/LIT methods/props are not available